client/app/views/components/pipelineView.js

```diff
@@ -17,10 +17,18 @@ var calcLastRun = function(pipeline) {
   return _.assign({}, pipeline, { finishedAt: lastRun(pipeline) });
 };
 
+var setPipelineId = function(pipeline) {
+  var pipelineId = function(name) {
+    return name.toLowerCase().replace(' ', '-');
```

Comment:
@viniciusgama doesn't the ruby API return an id attribute? I see that the fixtures in the JSON files doesn't have it but the example.yml does include the id attribute there.